### DISCUSSION TITLE
Generating more than 1 prompt at once

### DISCUSSION THREAD

**Discussion by RadzorD** [lllyasviel/Fooocus#2677] [Q&A] [answered]
I am interested in the issue of generating images immediately from several prompts.

For example, I have 50 ready-made prompts, and I would like them to be automatically generated in turn, instead of constantly changing the prompt myself.

Maybe this question has already been asked, but I missed it. Thank you for your help.

**Comment by ILikeToasters** [comment, marked as answer]
You can now do that with wildcards. Create a wildcard file for example prompts.txt in the Fooocus wildcard folder and put each prompt on a new line in the text file. Then you can just put `__prompts__` into the prompt and it will use the wildcard. But you also want to enable this  option in the debug section and be sure to set it to generate for the number of prompts in the file. 
<img width="170" alt="Screenshot 2024-03-30 231613" src="https://github.com/lllyasviel/Fooocus/assets/4357657/71243454-55cf-430d-8e0f-d4179fb9581d">

**Reply by mashb1t** [reply]
^ exactly this, @ILikeToasters thanks a lot!
